backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -2589,3 +2589,39 @@ databaseChangeLog:
                     name: session
                     type: jsonb
                     remarks: The HTTP session information for this request (only applies to REST requests).
+  - changeSet:
+      id: add-text-message-status-table
+      author: nicholas.a.scialli@omb.eop.gov
+      comment: Add table to track the delivery status of text messages
+      changes:
+        - createTable:
+            tableName: text_message_status
+            remarks: A table to store the various statuses of each text message
+            columns:
+              - column:
+                  name: internal_id
+                  type: uuid
+                  remarks: The internal database identifier for this entity
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: text_message_sent_internal_id
+                  type: uuid
+                  remarks: A foreign key to a text message sent
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk__internal_id__text_message_sent
+                    references: text_message_sent
+              - column:
+                  name: status
+                  type: text
+                  remarks: The status of the text message as reported by Twilio
+                  constraints:
+                    nullable: false
+              - column: *created_at_column
+              - column: *created_by_column
+              - column: *updated_at_column
+              - column: *updated_by_column
+        - sql: |
+            GRANT SELECT ON ${database.defaultSchemaName}.text_message_status TO ${noPhiUsername};
```

Comment:
Liquibase won't be able to determine an inverse for this raw SQL, so this changeset can't be rolled back without an explicit `rollback` instruction block.

Reply:
It shouldn't be needed for this particular migration since the migration adds a table. Necessarily, any access to that table will be gone if it's dropped on rollback

Reply:
Sorry, what I mean to say is that the bit of raw SQL keeps the _entire changeset_ from being rolled back:

```
nathancarter@kraut backend % ./gradlew liquibaseRollbackCount -PliquibaseCommandValue=1
Starting Liquibase at Tue, 03 Aug 2021 11:00:41 EDT (version 3.10.3 #32 built at Mon Oct 12 07:24:42 UTC 2020)         [25/375]
11:00:41.667 INFO  [liquibase.lockservice.StandardLockService]: Successfully acquired change log lock
11:00:42.034 INFO  [liquibase.changelog.StandardChangeLogHistoryService]: Reading from simple_report.databasechangelog
11:00:42.131 INFO  [liquibase.changelog.visitor.RollbackVisitor]: Rolling Back Changeset:classpath:/db/changelog/db.changelog-m
aster.yaml::add-text-message-status-table::nicholas.a.scialli@omb.eop.gov
11:00:42.143 INFO  [liquibase.lockservice.StandardLockService]: Successfully released change log lock
11:00:42.157 ERROR [liquibase.integration.commandline.Main]: Unexpected error running Liquibase: No inverse to liquibase.change
.core.RawSQLChange created
```

The `text_message_status` table will still exist after attempting this operation.

Reply:
_oh_. That's not great, I think we have a bunch of these through the changelog

Reply:
You bet we do 🙃